bindings/python/requirements.txt

```diff
@@ -1,3 +1,6 @@
 # Apache License 2.0
 
+open-space-toolkit-core~=3.0
+open-space-toolkit-mathematics~=3.0
+open-space-toolkit-physics~=6.0
```

Comment:
Shouldn't we rely on `open-space-toolkit-astrodynamics~=9.1` to define the related deps?

Reply:
I guess it's to keep the symmetry w/ te C++h side?

Reply:
(feel free to resolve if so)

Reply:
Right. So the reason we're adding this is because:
- For minor patches, let's say we add a feature to OSTk mathematics that we would like to use in OSTk physics. It shouldn't be necessary to bump physics, just to bump mathematics. 
For the packages that are directly used in library, we should import them explicitly, and this also allows us to have parity with the C++.
Let me know what you think.

Reply:
I haven't been following closely enough what the team has done on that front, I defer to you/Remy/Antoine/Kyle.

Reply:
No worries, thanks lucas!